IBPSA/Fluid/Movers/BaseClasses/FlowMachineInterface.mo

```diff
@@ -642,21 +642,15 @@ equation
     else
       eta_internal = cha.efficiency(per=per.efficiency, V_flow=V_flow, d=etaDer, r_N=r_N, delta=delta);
     end if;
-    if per.powerOrEfficiencyIsHydraulic then
-      P_internal=WFlo/IBPSA.Utilities.Math.Functions.smoothMax(
-                        x1=eta_internal, x2=1E-2, deltaX=1E-3);
-    else
-      P_internal=WHyd/IBPSA.Utilities.Math.Functions.smoothMax(
-                        x1=eta_internal, x2=1E-2, deltaX=1E-3);
-    end if;
+    P_internal=WFlo/IBPSA.Utilities.Math.Functions.smoothMax(
+                      x1=eta_internal, x2=1E-2, deltaX=1E-3);
```

Comment:
Such a change in code needs an entry in the model's revision notes.

Reply:
Greetings, Michael. This PR is still a draft. I created it so that I don't check the todo boxes in the issue post out of thin air. I will add the change logs in due course.